test/common/ssl/test_data/README.md

```diff
@@ -11,6 +11,12 @@ There are 4 identities:
 - **SAN With DNS**: It has the certificate *san_dns_cert.pem*, which is signed
   by the **CA** using the config *san_dns_cert.cfg*. The certificate has SAN
   field of DNS type. *san_dns_key.pem* is its private key.
+- **Fake CA**: Fake Certificate Authority used to validate verification logic.
+  It has the self-signed certificate *fake_ca_cert.pem"*. *fake_ca_key.pem" is
+  its private key.
+- **Self-signed**: The self-signed certificate *selfsigned_cert.pem" (in reality
```

Comment:
Actually, maybe it's better to self-sign the certificate. We don't need fake CA in this commit (though, I need it in subsequent changes).